### DISCUSSION TITLE
Proposal for Increasing Avalanche Governor Limits

### DISCUSSION THREAD

**Discussion by Micheldgit** [wormhole-foundation/wormhole#3634] [Governance] [answered]
Avalanche has been experiencing significant growth and increased transactional activity. This rise in usage has led to instances where the current governor limits have been reached in the last days (including today) hindering users and applications on the platform.

- The existing daily governor limit for on Avalanche is set at just $1 million, which is one of lowest among wormhole chains despite Avalanche being top 7 by market cap.
- Average daily volume for the last month has been approximately $475,000, which is a 4x increase from the average of $120k in the 6 months prior. This is also nearly 50% of the current maximum daily limit.
- Trend will increase significantly with the imminent launch of new application like [C3](https://c3.io/) and others using Avalanche as their CCTP hub.

We propose an increase in Avalanche’s Governor Limit from the current **$1 million/ $100k** to **$5 million / $500k**. This adjustment will provide sufficient headroom for the expected increase in transactional volume and activity.

**Comment by claudijd** [comment, marked as answer]
A PR to increase the limits as proposed: https://github.com/wormhole-foundation/wormhole/pull/3635

**Comment by aadam-10** [comment]
live as of [v2.23.32](https://github.com/wormhole-foundation/wormhole/releases/tag/v2.23.32)
